ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -91,7 +91,22 @@ azure_connection::delete_object(const std::string &object_key) const
 }
 
 int
-azure_connection::get_object(const std::string &path) const
+azure_connection::read_object(
```

Comment:
add a comment describing what the function does

Reply:
resolved https://github.com/wiredtiger/wiredtiger/pull/8738/commits/81b571a5438a27a5d569b49d4602a7878b0c46f5